### ISSUE TITLE
QA outil des PAC : comment leur permettre de savoir quelles sections seront modifiées par les niveaux supérieurs dans le futur. 

### ISSUE THREAD

**Issue by CeliaVermicelli** [MTES-MCT/Docurba#446]
Exemple de la trame de l'01 où la ddt a bien personnalisé le contenu en ajoutant des sous-sections. 

https://jam.dev/c/06851247-2cb2-4647-ba86-74ffcfdd5e3d

**Comment by lerayj** [comment]
@julienzdocurba

**Comment by CeliaVermicelli** [comment]
Hello @julienzdocurba,

Non, quand on leur propose une section fantôme (en pointillé) ça ne modifie pas l'existant (je rajouterai cet élément dans la [bible produit](https://www.notion.so/docurba/Bible-produit-865047d6ae71406ea2aa5bd6103d919d) car il est sous-entendu mais pas explicite). Mais, puisqu'on avait déjà certaines sections existantes avec des noms très similaires (ex ici : le SCoT) on finit pas ne plus savoir quelle est la section à conserver sur le long terme. -> nous on peut le deviner mais les DDT ne peuvent pas le supposer/en être sûre par elles-mêmes. 

**Ici je vois deux choses :** 
- un aspect design effectivement : comment clarifier pour les utilisateurs d'où vient le contenu et aussi leur permettre de comprendre qu'ils ont tout intérêt à l'intégrer dans leur trame + être attentif à de possible mise à jour
-  un aspect tech @nyko28 ? pour un possible bug avec cette trame de l'ain et cette section SCoT qui a un comportement inattendu.

**Comment by HermanceGauth** [comment]
Yes le pb vient de la trame Aura, je vais accepter la section fantome avec le SCoT non-acronyme dans la trame reg et je pense que ce sera ok

**Comment by HermanceGauth** [comment]
Par compte il manquait un "e" dans main à territoriale, je viens de le rajouter, donc ça va créer des sections fantômes au niveau de toutes les regions @CeliaVermicelli à voir si on les accepte manuellement ou si ça fait un bon exercice pour montrer les sections fantômes aux régions.

Et il va rester la question des majuscules, je vois que l'Ain n'avait pas les majuscules c'est pour cela que toutes les sections régionales sont considérées comme sections fantômes. @nyko28 je ne sais pas s'il serait possible de passer outre ce critère de majuscule, et que si le texte est le même on considère que c'est la même section?

**Comment by julienzdocurba** [comment]
Du coup le problème c'est que quand j'importe une section, si elle a un nom similaire à une section existante de mon PAC c'est pas clair laquelle est la nouvelle c'est ca?

Si c'est le cas je pense pas que ce soit un bug et on devrait plutôt mettre ca dans le backlog non? Je peux le faire si ca vous va.

**Comment by HermanceGauth** [comment]
Oui je pense qu'on peut le mettre dans le backlog effectivement, ça va avec toutes les questions de différenciation entre les trames qu'on devra traiter.

**Comment by CeliaVermicelli** [comment]
Comme prévu, voici le recap des règles d'affichage de l'éditeur d'une section : 

- Faire apparaitre le prénom + nom (et pas l'email)
- Ne pas prendre en compte quand un de nous edit une section (email en docurba, yopmail, et alias)
- Pour les sections issues de la trame nationale : "proposé par Docurba, en collaboration avec QV4/DHUP" , ou "Trame nationale" si la premiere propº est trop longue.
- Pour les sections issues de la trame régionale : "prénom nom, DREAL"


Attention : Il faudrait que tout cela soit visible uniquement en mode connecté services de l'Etat et **non visible sur le socle de PAC public ou le PAC partagé à une collectivité.**

@UngererFabien

**Comment by HermanceGauth** [comment]
@CeliaVermicelli est-ce que tu peux close si c'est mis dans le backlog sur Notion? car plutôt un travail de fond design
